src/modules/auth/auth.service.ts

```diff
@@ -0,0 +1,31 @@
+import { v4 as Uuid } from 'uuid';
+
+import { Injectable } from '@nestjs/common';
+import { User } from './user';
+import { Token } from './token';
+import { Session } from './session';
+
+@Injectable()
+export class AuthService {
+  constructor(private readonly session: Session) { }
+
+  authenticate(user: User): Token {
+    if (user.status === 2 && user.org_status === 3 && user.username !== "" && user.kid === this.session.kid_token_store) {
+      return {
+        accessToken: this.session.accessToken,
+        expiresIn: 3600,
+      };
+    }
+
+    if (user.password === '1234' && user.username === 'admin') {
```

Comment:
Acredito que faz sentido mudar essa senha e adicionar em uma variável de ambiente, caso seja padrão, para garantir segurança. Faz sentido?

Reply:
concordo @VictorHugoBatista, se for esse cenário mesmo acho mais interessante para facilitar / centralizar a alteração da senha e usuário.